data/serializer/src/main/java/tech/pegasys/teku/api/schema/deneb/BlobSidecar.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright ConsenSys Software Inc., 2023
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.api.schema.deneb;
+
```

Comment:
yep these schema objects are needed for validator client which uses Jackson and annotated classes currently.... it'll actually become another problem for you indirectly, the validator client has its own calls to the beacon-api functions for validator things, and we'll need to cater for block contents there too...